exotic.py

```diff
@@ -889,7 +926,7 @@ def contextupdt(times=None, airm=None):
 
 # -- LIGHT CURVE MODEL -- ----------------------------------------------------------------
 def lcmodel(midTran, radi, am1, am2, theTimes, theAirmasses, plots=False):
-    sep, ophase = time2z(theTimes, inc, midTran, semi, planetPeriod, eccent)
+    sep, ophase = time2z(theTimes, pDict['inc'], midTran, pDict['aRs'], pDict['pPer'], pDict['ecc'])
```

Comment:
pDict will now be scattered throughout the code rather than creating a new variable for each one. These remain static and already exist, no need to create new variables.